packages/client/test/unit/EncryptionUtil.test.ts

```diff
@@ -53,7 +51,7 @@ describe('EncryptionUtil', () => {
             nextEncryptionKey: nextKey
         })
         EncryptionUtil.decryptStreamMessage(streamMessage, key)
-        expect(streamMessage.getSerializedContent()).toStrictEqual('{"foo":"bar"}')
+        expect(binaryToUtf8(streamMessage.getSerializedContent())).toStrictEqual('{"foo":"bar"}')
```

Comment:
Simpler assertion by checking the `expected` as-is:

```suggestion
        expect(streamMessage.getSerializedContent()).toStrictEqual(utf8ToBinary('{"foo":"bar"}'))
```

Reply:
added comment